addons/mail/static/src/components/discuss_sidebar_category/discuss_sidebar_category.xml

```diff
@@ -25,17 +25,11 @@
             </div>
             <div class="o_DiscussSidebarCategory_content">
                 <t t-if="category.isOpen">
-                    <t t-if="category.isAddingItem">
+                    <t t-if="category.itemNewInputView">
                         <div class="o_DiscussSidebarCategory_newItem">
```

Comment:
(note about existing code) strange to have a div between the view record t-if and the view record itself, but I guess it's acceptable if extra style is needed in this case only

Reply:
it's indeed just for styling